scripts/create-release.sh

```diff
@@ -4,8 +4,8 @@ set -o errexit
 echo "Bumping version from: $(scripts/print-version.sh $1 $2)"
 VERSION="$(scripts/bump-version.sh $1 $2)"
```

Comment:
Using relevantly named variables for script params would improve readability a bit.